common/chat/helper.ts

```diff
@@ -90,6 +91,21 @@ const checkIfSubcourseParticipation = async (participants: string[]): Promise<bo
     return result.length > 0;
 };
 
+const getParticipantsForSubcourseGroupChat = async (subcourse: Subcourse, instructor: User) => {
```

Comment:
Also, the word participant might be misleading here. Maybe members would be better to differentiate between subcourse and chat participants?